### ISSUE TITLE
project_polly.gltf is not displayed

### ISSUE THREAD

**Issue by cx20** [Reon90/redcube#7]
Project "Polly" is a model created to introduce all the new features of glTF 2.0.
https://github.com/KhronosGroup/glTF-Blender-Exporter/tree/master/polly

However, RedCube.js does not seem to be able to display Polly yet.
[RedCube.js + project_polly.gltf  result:](http://jsdo.it/cx20/kivX) 
http://jsdo.it/cx20/kivX
```
redcube.js:8 
Uncaught (in promise) TypeError: Cannot read property 'matrixWorld' of undefined
    at d.getJointMatrix (redcube.js:8)
    at d.setSkin (redcube.js:8)
    at Be.buildPrim (redcube.js:8)
    at Array.map (<anonymous>)
    at Be.buildNode (redcube.js:8)
    at Array.forEach (<anonymous>)
    at Be.buildNode (redcube.js:8)
    at json.scenes.(anonymous function).nodes.forEach.e (https://rawcdn.githack.com/cx20/gltf-test/a9cda755ff1c6656c5b3797b8747153e9e630947/libs/redcube/20181228/redcube.js:8:1727400)
    at Array.forEach (<anonymous>)
    at Be.buildMesh (redcube.js:8)
```

Below is the display result of Three.js and GLTFLoader.
[Three.js + project_polly.gltf result:](http://jsdo.it/cx20/qugw) 
http://jsdo.it/cx20/qugw
![image](https://user-images.githubusercontent.com/1833979/50548671-03cd7380-0c94-11e9-9d78-d06b410c9da6.png)

**Comment by Reon90** [comment]
@cx20 Was fixed in 2.0.5

**Comment by cx20** [comment]
I confirmed that Polly will be displayed in the latest version of RedCube.js.
[RedCube.js + project_polly.gltf result:](http://jsdo.it/cx20/OnVk)
![image](https://user-images.githubusercontent.com/1833979/50579826-e82cbd80-0e8b-11e9-9094-954a465ae57c.png)
